src/utils/helpers/list/mapColumnsByRole.ts

```diff
@@ -106,7 +106,7 @@ export const useMapColumnsByRole = () => {
       sortName: '',
       ColumnComponent: ReviewerActionCell,
       hideMobileLabel: true,
-      hideOnMobileTest: (application) => !application.reviewerAction,
+      hideOnMobileTest: (_) => true,
```

Comment:
Always hide the action cell in the normal flow now, as we handle it as a special case (as above)